SPECS/kernel/config_aarch64

```diff
@@ -589,31 +635,74 @@ CONFIG_ACPI_IPMI=m
 CONFIG_ACPI_HOTPLUG_CPU=y
 CONFIG_ACPI_THERMAL=y
 CONFIG_ARCH_HAS_ACPI_TABLE_UPGRADE=y
-# CONFIG_ACPI_TABLE_UPGRADE is not set
+CONFIG_ACPI_TABLE_UPGRADE=y
```

Comment:
This config has been identified by the kernel config hardening checker and should be disabled

Reply:
Discussed offline. May be required for boot. Will prune down in future PR.